src/periscope.ts

```diff
@@ -34,8 +34,11 @@ type DisposablesMap = {
   query: vscode.Disposable[]
 };
 
+// Allow other commands to access the QuickPick
+let activeQP : vscode.QuickPick<AllQPItemVariants> | undefined;
+let previousActiveEditor: vscode.TextEditor | undefined;
```

Comment:
I don't like these global variables, let me know if you have a better solution.

Reply:
Should be fine since these are ESM imports, they will only be scoped to the file module.